cpp/src/arrow/compute/row/compare_internal.h

```diff
@@ -92,7 +98,8 @@ class ARROW_EXPORT KeyCompare {
   static uint32_t NullUpdateColumnToRowImp_avx2(
       uint32_t id_col, uint32_t num_rows_to_compare, const uint16_t* sel_left_maybe_null,
       const uint32_t* left_to_right_map, LightContext* ctx, const KeyColumnArray& col,
-      const RowTableImpl& rows, uint8_t* match_bytevector);
+      const RowTableImpl& rows, uint8_t* match_bytevector,
+      bool are_cols_in_encoding_order);
```

Comment:
Nit: output parameter (`uint8_t* match_bytevector`) should be last.

Reply:
Done.